dedupe/variables/ilcs.py

```diff
@@ -0,0 +1,22 @@
+import ilcs_parser
+from parseratorvariable import ParseratorType
+
+CITATION = (
+    ('chapter', ('Chapter',)),
+    ('act prefix', ('ActPrefix')),
+    ('section', ('Section')),
+    ('subsection', ('SubSection')),
+    ('attempted', ('Attempted'))
+)
+
+
+class ILCSType(ParseratorType):
+    type = 'ILCS'
```

Comment:
Do we need to define different predicates here, or are the default `ParseratorType` predicates adequate? I don't fully understand what's going on in the code block below that initializes predicates:

https://github.com/dedupeio/parseratorvariable/blob/bb8808b05418300bcf42d88bc4e8570a65011ffa/parseratorvariable/__init__.py#L58-L69

Reply:
should be fine to start with. (basically we define predicates on the the parsed parts)